variables.tf

```diff
@@ -46,7 +46,14 @@ variable "retention" {
   default     = 30
 }
 
+variable "provision_trigger" {
+  type        = bool
+  description = "Whether or not to create a lambda trigger from an SNS topic"
+  default     = "false"
+}
+
 variable "sns_topic_arn" {
-  type        = string
+  type        = list(string)
```

Comment:
Are we using this anywhere at the moment that might not expect this to be a list?

Reply:
This is a brand new variable, so we aren't using it anywhere that I haven't put it.